index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
```

Comment:
❗️ 수정요청

html lang 속성은 스크린리더, 검색 엔진등에게 해당 사이트가 어떤 언어 기반으로 작성되었는지 설명하는 속성입니다.
해당 속성에는 이미 특정 언어들을 의미하는 예약어가 들어가야 합니다. 예를들어 많이 사용되는 en은 english 즉 영어를 의미합니다.
아래처럼 한국어를 의미하는 ko를 넣어주세요~

```suggestion
<html lang="ko">
```